CMakeLists.txt

```diff
@@ -16,6 +16,24 @@ list( APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/etc/cmake" )
 include( BBVendor )
 include( TargetBMQStyleUor )
 
+if(DEFINED VCPKG_TOOLCHAIN)
+  find_package(bsl CONFIG PATH_SUFFIXES share/bde REQUIRED)
+  find_package(bal CONFIG PATH_SUFFIXES share/bde REQUIRED)
+  find_package(bdl CONFIG PATH_SUFFIXES share/bde REQUIRED)
+  find_package(nts REQUIRED)
+  find_package(ntc REQUIRED)
+  find_package(benchmark CONFIG REQUIRED)
+  find_package(GTest CONFIG REQUIRED)
+  find_package(ZLIB REQUIRED)
+
+  add_library(benchmark ALIAS benchmark::benchmark)
+  add_library(zlib ALIAS ZLIB::ZLIB)
```

Comment:
Are the `pkgname::pkgname` target names a VCPKGism?  If so, how come it's just these, and not GTest for instance?  I'd like a short comment right above this line explaining exactly why this is necessary.

Reply:
This is how many vcpkg packages choose to export components. It's a common pattern among other uses of CMake. `GTest` as you can tell is not necessary, must have mistakenly added it while I was testing this.

Reply:
Resolved in comment.